docs/operation/volume_management/storageclass_creation.rst

```diff
@@ -0,0 +1,51 @@
+StorageClass Creation
+=====================
+
+**MetalK8s** uses **StorageClass** to configure how **Volumes** are formatted
+and mounted.
+This section hightlights how to create a Storageclass using the **CLI**.
+
+#. Create a StorageClass manifest.
+
+   You can define a new **StorageClass** using the following template:
+
+   .. code-block:: yaml
+
+       apiVersion: storage.k8s.io/v1
+       kind: StorageClass
+       metadata:
+         name: <storageclass_name>
+       provisioner: kubernetes.io/no-provisioner
+       reclaimPolicy: Retain
+       volumeBindingMode: WaitForFirstConsumer
+       mountOptions:
+         - rw
+       parameters:
+         fsType: <filesystem_type>
+         mkfsOptions: '["-m", "0"]'
```

Comment:
Aren't these options also FS-specific? For example, that `-m` is the percentage of reserved blocks in ext4, but some kind of message file in FAT.

Reply:
Yes they are, we should probably make them "generic" as well, and also mention that the field is optional.

Reply:
Yes they seem to be `File system` specific but we only support `XFS` and `EXT4` mainly so this would still work i guess. :)

Reply:
Actually no :stuck_out_tongue: 

We already have a wide difference in how to specify `ext4` and `xfs` option.
For instance, even setting the UUID:
- `-U <UUID>` for `ext4`
- `-m uuid=<UUID>` for `xfs`

So `-m 0` isn't gonna work for `xfs` (for the record, reserved blocks are not handled the same way in XFS and ext4, see [here](https://www.spinics.net/lists/xfs/msg30770.html)).

So we definitely wants something generic here, such as `<mkfs_options>`.